google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/data/v2/stub/metrics/BuiltinMetricsTracer.java

```diff
@@ -358,6 +372,9 @@ private void recordAttemptCompletion(@Nullable Throwable status) {
     attemptLatenciesHistogram.record(
         convertToMs(attemptTimer.elapsed(TimeUnit.NANOSECONDS)), attributes);
 
+    remainingDeadlineHistogram.record(
+        operationTimeout.toMillis() - operationTimer.elapsed(TimeUnit.MILLISECONDS), attributes);
```

Comment:
If we somehow failed to set operationTimeout, this will break. I think we should verify operationTimeout.toMillis() - operationTimer.elapsed is >= 0 before recording it. If it's < 0, log an error.